content/javascript/concepts/promise/promise.md

```diff
@@ -81,3 +81,40 @@ promise
 
 // Output: Success: The promise was fulfilled!
 ```
+
+#### Async Await
+
+Starting from ES2017, JavaScript introduced the keywords "async" and "await", which is used to write promises in a cleaner way. This can be used as an alternative to other verbose solutions, such as promise chaining.
```

Comment:
```suggestion
Starting from ES2017, JavaScript introduced the keywords `async` and `await`, which are used to write promises in a cleaner way. This can be used as an alternative to other verbose solutions, such as promise chaining.
```